docs/src/starknet/verify.md

```diff
@@ -0,0 +1,39 @@
+# Verifying Contracts
+
+## Overview
+
+Starknet Foundry `sncast` supports verifying Cairo contract classes with the `sncast verify` command by submitting the source code to a selected verification provider.
+
+For detailed CLI description, see [verify command reference](../appendix/sncast/verify.md).
+
+> ⚠️ **Warning**
+> Please be aware that submitting the source code means it will be publicly exposed through the provider's APIs.
```

Comment:
I think it's also worth explaining here what happens to the code on the backend (how it can be used)